tests/platform_tests/api/test_sfp.py

```diff
@@ -421,13 +446,18 @@ def test_get_tx_power(self, duthosts, enum_rand_one_per_hwsku_hostname, localhos
         skip_release_for_platform(duthost, ["202012"], ["arista", "mlnx"])
 
         for i in self.sfp_setup["sfp_test_port_indices"]:
+            info_dict = sfp.get_transceiver_info(platform_api_conn, i)
+
+            if not self.is_xcvr_optical(info_dict):
+                logger.info("test_get_tx_power: Skipping transceiver {} (not supported on this platform)".format(i))
```

Comment:
the log message "not support on this platform" is incorrect. it should be something like "not support on DAC cable" or something like [https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_sfp.py#L387](https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_sfp.py#L387)